Misc/NEWS.d/next/Library/2017-11-30-20-38-16.bpo-32186.O42bVe.rst

```diff
@@ -0,0 +1,3 @@
+Fixed hang of all threads when using io.FileIO with inaccessible NFS server.
+Now only the thread accessing the problematic storage will hang.  Patch by
```

Comment:
"io.FileIO" is unclear. IMHO it's better here to mention the impacted methods, so read and readall, and mention the GIL. I propose:

io.FileIO.readall() and io.FileIO.read() now releases the GIL when getting the file size. Fixed hang of all threads with inaccessible NFS server. Patch by Nir Soffer.

Reply:
I'm happy with your suggestion, updating :+1: